src/chords_generation/daria.py

```diff
@@ -105,14 +98,14 @@ def get_chords_daria(sounds: List[music.Sound],
                 max_chord = c
 
         t_c.append(music.Chord(max_chord.note,
-                               duration=half_bar_len/4,
+                               duration=constants.MIN_CHORD_DURATION,
                                kind=max_chord.kind))
         last_chord_level = max_chord.level
-        i += half_bar_len
+        i += constants.MIN_CHORD_DURATION
```

Comment:
`i` should also be changed by width of the window

Reply:
Done